lib/klp.pm

```diff
@@ -45,10 +46,14 @@ sub install_klp_product {
         $lp_module  = 'SLE-Live-Patching';
     }
 
-    #install kgraft product
-    zypper_call("ar http://download.suse.de/ibs/SUSE/Products/$lp_module/$version/$arch/product/ kgraft-pool");
-    zypper_call("ar $release_override http://download.suse.de/ibs/SUSE/Updates/$lp_module/$version/$arch/update/ kgraft-update");
-    zypper_call("ref");
+    if ($livepatch_repo) {
+        bmwqemu::fctwarn("pev: IF livepatch_repo: '$livepatch_repo'");
```

Comment:
It looks like  personal debug/warning message.

Reply:
Thx! Fixed.